lib/universal/mapper.js

```diff
@@ -70,12 +70,16 @@ exports.track = function(track, settings) {
   var result = createPageDataForm(track, createCommonGAForm(track, settings));
   var library = track.proxy('context.library');
   var screenName = track.proxy('properties.screenName');
+  var url = track.proxy('context.page.url') || track.proxy('properties.url');
+  var library = track.proxy('context.library');
+  var screenName = track.proxy('properties.screenName');
```

Comment:
We already do this for screen calls https://github.com/segment-integrations/integration-google-analytics/blob/master/lib/universal/mapper.js#L51

This change  is relevant specifically  only for track calls.